extensions/resteasy-qute/runtime/src/main/java/io/quarkus/resteasy/qute/runtime/ResteasyTemplate.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.resteasy.qute.runtime;
+
+import javax.ws.rs.container.ResourceInfo;
+
+import org.jboss.resteasy.core.ResteasyContext;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.Template;
+import io.quarkus.qute.TemplateInstance;
+
+public class ResteasyTemplate {
```

Comment:
I'm not sure about the name... maybe just `RestTemplate`? IDK maybe it's OK. But it should reside in the `io.quarkus.resteasy.qute` package as it's a public API. Plus it wouldn't hurt to make it final with a private constructor.

Reply:
OK changed.